web/src/widget/image.rs

```diff
@@ -15,13 +15,16 @@ use std::{
 /// ```
 /// # use iced_web::Image;
 ///
-/// let image = Image::new("resources/ferris.png");
+/// let image = Image::new("resources/ferris.png", "Ferris the crab");
 /// ```
 #[derive(Debug)]
 pub struct Image {
     /// The image path
     pub handle: Handle,
 
+    /// The alt text of the image
+    pub alt: String,
```

Comment:
images should always have alt text unless they are purely presentational, e.g. a background image. im not sure the best way to go about making this not necessary for those sorts of images, and this feels like a good default

Reply:
Could always be passed an empty string? Not sure how that works on screen readers and such though.

Reply:
it could. i wasn't sure that implementing some sort of length requirement would be the correct thing to do since it would end up just being arbitrary. and i feel like most people ignore accessibility out of ignorance rather than malice, so having a reminder that `alt` text is required would be enough for most cases

Reply:
but it seems like adding them as builder methods as @hecrj suggested will result in less empty alt tags since people will have to opt in

Reply:
We can make `alt` text mandatory once we tackle accessibility in the native side of the ecosystem.